float64 lin_vel
float64 ang_vel
float64 accel
geometry_msgs/Point goal
float64[] lidar
geometry_msgs/Point[] pedestrians
float64 goal_distance
float64[] grid_map